doc_swift_ui_gallery/swift_ui_gallery/swift_ui_gallery/layouts/HStackExamples.swift

```diff
@@ -0,0 +1,227 @@
+import SwiftUI
+
+struct HStackLeftAlignedPage : View {
+    var body: some View {
+        NavigationStack {
+            HStack() {
+                Text("First")
+                Text("Second")
+                Text("Third")
+                Spacer()
```

Comment:
While this is a legitimate `HStack` example, it doesn't really reflect the name of "Left Aligned". This is an example of an `HStack` with an expanding child.

Isn't there an alignment property or alignment modifier that applies to `HStack`? If so, this example should work that way. The same goes for all other examples. Otherwise, all we're really testing is the movement of a `Spacer()` to different positions in the child list.

Reply:
As per the Apple docs `HStack` uses `VerticalAlignment` and `VStack` uses `HorizontalAlignment`.
Ref: https://developer.apple.com/documentation/swiftui/aligning-views-within-a-stack#Customize-stack-alignment

So if we need to use `HStack` with `HorizontalAlignment`, one way is to wrap the `VStack` items inside `HStack` and then use it. It is also shown in the docs here: https://developer.apple.com/documentation/swiftui/horizontalalignment

Here in the above example individual items of `HStack` are `VStack` elements which have been given, (.leading, .center, .trailing) alignment. Other way is to use the `Spacer()`

Reply:
I'm not sure my point came across. You've labeled this example as "left aligned", however you're not really demonstrating alignment. You're demonstrating how a spacer works. A spacer can go anywhere, and the way a spacer works is different from how alignment works.

If you want to demonstrate `HStack` alignment, or `VStack` alignment, you should use the properties that are built into those views for alignment.

If you want to demonstrate how to use spacers, then the associated name should be about spacers, rather than alignment.

In this case, alignment is more important than spacers, so we should use the alignment properties, and keep the example names as-is.